tests/unit/policy/test_policyValidator.js

```diff
@@ -71,6 +72,98 @@ describe('Policies validation - Version', () => {
     });
 });
 
+describe('Policies validation - Principal', () => {
+    [
+        {
+            name: 'an account id',
+            value: { AWS: '111111111111' },
+        },
+        {
+            name: 'anonymous user AWS form',
+            value: { AWS: '*' },
+        },
+        {
+            name: 'an account arn',
+            value: { AWS: 'arn:aws:iam::111111111111:root' },
+        },
+        {
+            name: 'multiple account id',
+            value: {
+                AWS: ['111111111111', '111111111112'],
+            },
+        },
+        {
+            name: 'multiple account arn',
+            value: {
+                AWS: [
+                    'arn:aws:iam::111111111111:root',
+                    'arn:aws:iam::111111111112:root',
+                ],
+            },
+        },
+        {
+            name: 'anonymous user as string',
+            value: '*',
+        },
+        {
+            name: 'user arn',
+            value: { AWS: 'arn:aws:iam::111111111111:user/alex' },
+        },
+        {
+            name: 'multiple user arns',
+            value: {
+                AWS: [
+                    'arn:aws:iam::111111111111:user/alex',
+                    'arn:aws:iam::111111111111:user/thibault',
+                ],
+            },
+        },
+        {
+            name: 'role arn',
+            value: {
+                AWS: 'arn:aws:iam::111111111111:role/dev',
+            },
+        },
+        {
+            name: 'multiple role arn',
+            value: {
+                AWS: [
+                    'arn:aws:iam::111111111111:role/dev',
+                    'arn:aws:iam::111111111111:role/prod',
+                ],
+            },
+        },
+        {
+            name: 'saml provider',
+            value: {
+                Federated:
+                    'arn:aws:iam::111111111111:saml-provider/mysamlprovider',
+            },
+        },
+        {
+            name: 'with backbeat service',
+            value: { Service: 'backbeat' },
+        },
+    ].forEach(test => {
```

Comment:
These tests are useful, though they should be completed with some failure tests to be really valuable.